hr_timesheet_sheet_attendance/readme/CONTRIBUTORS.rst

```diff
@@ -2,6 +2,6 @@
 * Shruti Singh <shruti.singh@bizzappdev.com>
 * Chirag Parmar <chirag.parmar@bizzappdev.com>
 * Naglis Jonaitis <naglis@versada.eu>
+* Dhara Solanki <dsolanki@initos.com>
 * `Tecnativa <https://www.tecnativa.com>`_:
-
-  * Ernesto Tejeda
+* Ernesto Tejeda
```

Comment:
???

Reply:
> ???

Sorry, I did not get your question?

Reply:
Why this change?

Reply:
> Why this change?

I've modified it as per your suggestion can you please review it again?

Reply:
No, it's wrongly formatted, can't you just leave it as it was?